index.html

```diff
@@ -10,9 +10,10 @@
         <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="original trombones logo">
         <nav id="nav-bar">
             <a class="nav-link" href="#features">Features</a>
-            <a class="nav-link" href="#howitworks">How it works</a>
+            <a class="nav-link" href="#howitworks">How does it works</a>
             <a class="nav-link" href="#pricing">Pricing</a>  
```

Comment:
Please use symantic tag like ul & li  asThe semantic HTML tags help the search engines and other user devices to determine the importance and context of web pages.